fastify.js

```diff
@@ -411,13 +413,22 @@ function build (options) {
     }
 
     validateBodyLimitOption(opts.jsonBodyLimit)
-    var jsonBodyLimit = opts.jsonBodyLimit || _fastify._jsonBodyLimit
 
-    _fastify.after((notHandledErr, done) => {
+    _fastify.after(function afterRouteAdded (notHandledErr, done) {
       const path = opts.url || opts.path
       const prefix = _fastify._routePrefix
       const url = prefix + (path === '/' && prefix.length > 0 ? '' : path)
 
+      opts.url = url
```

Comment:
Since we support both `url` and `path`, should we add the path string to the final object with the prefix?
Basically a mirror of `url`.

Reply:
Could you make an example?

Reply:
Our `route` api supports both `url` and `path` in the [route declaration](https://github.com/fastify/fastify/blob/master/docs/Routes.md#full-declaration).
Given that, I think the user expects to find both `url` and `path` inside the `onRoute` hook.

Reply:
Allright